### ISSUE TITLE
Filepond stop handling files after error() function being called

### ISSUE THREAD

**Issue by Windman** [pqina/filepond#260]
Hello

### Summary
I've implemented my own Process function and then error() callback being called
'filepond' stop handling files.
...

### How to reproduce
Setup environment
"ngx-filepond": "^5.0.0"
"filepond": "^4.2.0" // ^3.9.0 same issue

app.component.html
```html
<file-pond #myPond 
    [options]="options" 
    (oninit)="pondHandleInit()"
    (onaddfile)="pondHandleAddFile($event)">
</file-pond>
```
app.component.ts
```typescript
import { Component, ViewChild } from '@angular/core';

@Component({
	selector: 'app-root',
	templateUrl: './app.component.html',
	styleUrls: ['./app.component.css']
})
export class AppComponent {
	@ViewChild('myPond') myPond: any;
	options: any;
	files: Map<string, any> = new Map<string, any>();
	
	constructor() {
		const processHandler = (
			fieldName,
			file,
			metadata,
			load,
			error,
			progress,
			abort
		) => {

			//load(file.name);
			error('upload-error');

			return {
				abort: () => {
					abort();
				}
			};
		};

		const pondOptions = {
			multiple: true,
			maxFiles: 5,
			maxParallelUploads: 1,
			dropOnPage: true,
			dropOnElement: false,

			server: {
				process: processHandler,
				revert: null
			}
		};

		this.options = pondOptions;
	}

	pondHandleInit() {
		console.log('FilePond has initialised', this.myPond);
	}

	pondHandleAddFile(event: any) {
		this.files.set(event.file.file.name, event.file);
	}
}
```
Select couple of files

### Expected behaviour
After error callback being called continue process files in the queue
...

### Additional information

...

| Environment  | Version
|------------- | -----------
| OS              |  Windows / Linux / ...
| Browser      | Chrome

**Comment by rikschennink** [comment]
Could be because the error callback is called immediately or that only one file can be uploaded at a time, I'll have to test this but am quite busy at the moment.

**Comment by rikschennink** [comment]
Fixed in version `4.3.0`
